doc-content/enterprise-only/openshift/rhpam73-prod-immutable-monitor-ref.adoc

```diff
@@ -175,15 +178,27 @@ for more information.
 ==== Pod Template
 
 
+===== Service Accounts
+
+Service accounts are API objects that exist within each project. They can be created or deleted like any other API object. Refer to the
+https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-service-accounts#dev-managing-service-accounts[Openshift documentation] for more
+information.
```

Comment:
Minor suggestion - to adopt IBMSG for more information links, change link title to section, and combine into one line.

Service accounts are API objects that exist within each project. They can be created or deleted like any other API object. For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-service-accounts#dev-managing-service-accounts[Managing Service Accounts] in the _Openshift documentation_.